sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -127,7 +127,18 @@ Licensed under the MIT License.
         </exclusion>
       </exclusions>
     </dependency>
-
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-tracing-opentelemetry</artifactId>
```

Comment:
Is this test dependency being used even when you are using mockito?

Reply:
Yes